DB/api/test_query_db.py

```diff
@@ -96,10 +98,12 @@ def test_classify(self):
 
```

Comment:
The reason this test is failing is because it is still stubbing out the old analysis library. You need to change it from stubbing out Analysis_3.run to stubbing out RMF.RandomForestModel.get_model and model.classify.